sample/sampleUploadApplication.php

```diff
@@ -0,0 +1,132 @@
+<?php
```

Comment:
Some code here is pretty similar to the other sample, so would be nice to put the common code in some reusable piece.